packages/lodestar/src/api/impl/beacon/beacon.ts

```diff
@@ -92,9 +92,9 @@ export class BeaconApi implements IBeaconApi {
 
   public getBlockStream(): LodestarEventIterator<SignedBeaconBlock> {
     return new LodestarEventIterator<SignedBeaconBlock>(({push}) => {
-      this.chain.on("processedBlock", push);
+      this.chain.emitter.on("processedBlock", push);
```

Comment:
since you are already refactoring chain emitting. Maybe rename event to just "block" and have another "head" which should help us implement standardized event stream api

Reply:
:+1: I was trying not to do too much in this PR, but I'll go ahead and rename these